spec/controllers/gallery/gallery_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe GalleryController, type: :controller do
+  let(:user) { create(:user) }
+  let(:album) { create(:album) }
+  let(:old_album) { create(:album, start_date: Time.new(1986,3,25)) }
```

Comment:
Space missing after comma.<br>Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.